fixtures/setup.json

```diff
@@ -1,23 +1,35 @@
 {
   "seed": 4,
   "keysToGen": 5,
+  "importKeys": [{"privateKey":"/8UoXvzGIU2H/TgTAmswx96m2dVCXkRqnIIlDhJrGCo=","sigType":2}],
   "preAllocGenKeys": [
     "10000000000",
     "10000000000",
     "10000000000",
     "10000000000",
+    "10000000000",
```

Comment:
`keysToGen` is still 5, so what does this do? Or is this allocating to the non-gen keys too (in which case please rename `preAllocGenKeys`)?

Reply:
It is allocating to all keys. I'll rename.

Reply:
renamed to `PreallocatedFunds`.